CryptoPrices/Sources/MyCoin/Sources/MyCoin/MyCoinView/MyCoinView.swift

```diff
@@ -43,8 +43,9 @@ private extension MyCoinView {
     var contentView: some View {
         ScrollView {
             CurrentPriceSection()
-            // TODO: - Add Chart and timeframe here
+            // TODO: - Add Chart here
             // TODO: - Remove dummy
+            TimeFrameSection()
```

Comment:
A quick suggestion, maybe your branch can be based on my PR [here](https://github.com/nimblehq/nimble-crypto-ios/pull/87) for completely cleanup this `// TODO: - Add Chart here` 😁

Reply:
Thanks. I fixed it at https://github.com/nimblehq/nimble-crypto-ios/pull/86/commits/e096d963ba508001f5aae3513b0af1dc28571083.